src/lib/matchers.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Collection of useful custom jasmine matchers for tests.
+ */
+export const customMatchers: jasmine.CustomMatcherFactories = {
```

Comment:
As long as the `matchers.ts` file is not imported by any source file and it will be only imported by spec files it should be fine.

What I recommend though: Store the filer under `core/testing` and name it `jasmine-matchers.ts`